test/libsolidity/semanticTests/array/copying/array_of_function_external_storage_to_storage_dynamic.sol

```diff
@@ -0,0 +1,43 @@
+contract C {
+    function testFunction1() public {}
+    function testFunction2() public {}
+    function testFunction3() public {}
+
+
+    function() external [3] externalArray0;
+    function() external [3] externalArray1 = [
+        this.testFunction1,
+        this.testFunction2,
+        this.testFunction3
+    ];
```

Comment:
I think we should test both static and dynamic arrays but the dynamic ones are more important - that's what the original issue was about.

Also the test name is wrong now - it says `dynamic` but uses static arrays.

Reply:
This PR had already been merged. So, I've added the change in #12417